### ISSUE TITLE
Controls are invisible in OLED low-backlight with pitch black theme

### ISSUE THREAD

**Issue by flaki** [nolanlawson/pinafore#1868]
I'm using Pinafore on my Pixel 3 which has an OLED screen. Because of this, I use the "pitch black" theme to avoid light leakage/extra battery consumption from the really-really-dark-grey backgrounds. The display colors in settings are set to "Adaptive".

In the night in a dark room or similar light-deprived scenarios the phone frequently lowers the screen brightness to a minimum which still leaves everything (toots, images) perfectly readable -- except the action icons below the tweets which basically disappear.

Imagine one of your "calibrate your display gamma" scenarios in games, where they say "make sure the icon on the middle is barely visible". Even on not-minimum-but-low-backlight the icons are not very well visible and one needs to increase the display backlight to quite high for them to show up.

A picture is worth more than a thousand words so I have a blurry gif that demonstrates the issue:
![Bug GIF-downsized_large](https://user-images.githubusercontent.com/2089432/92531805-4dbf7880-f22f-11ea-888d-d13dc63b1373.gif)

I'd be happy to contribute a fix if we can agree what could be the best way to fix these?

An idea I had would be medium-bright 1-2px outlines for these "inactive" icons?

**Comment by nolanlawson** [comment]
Thanks for opening the issue! The Pitch Black theme was specifically designed for OLED screens, but I don't have one so I'm not able to repro. Since this is an accessibility issue, I'd be happy to accept any PR that makes the icons visible.  :slightly_smiling_face:

**Comment by NickColley** [comment]
Fixed in #2324
